### ISSUE TITLE
Additional Resources

### ISSUE THREAD

**Issue by vmbrasseur** [DuaneOBrien/SpeakerKata#4]
I've been maintaining a list of [public speaking resources](https://github.com/vmbrasseur/10stepprogram/blob/master/resources.md) in the [repo](https://github.com/vmbrasseur/10stepprogram) Josh & I used for our speaker training slides & materials.

We might want something like that for SpeakerKata. If so, naturally we can feel free to use the resources.md file as a starting point. It'd need some [some organization](https://github.com/vmbrasseur/10stepprogram/issues/48).

**Comment by DuaneOBrien** [comment]
Would it be sufficient to help you refactor your resources and point to that?

**Comment by vmbrasseur** [comment]
Yeah, that could work. But I think it'd probably be better just to get it out of that repo. It's looking more and more like that talk was a one-time thing (in that incarnation; we each do our own versions now). The resources.md file should move somewhere more sustainable.

I'm happy to organise it and move it here, if there are no objections.

**Comment by vmbrasseur** [comment]
Hey @DuaneOBrien? Could you please assign this issue to me so I'm less likely to lose track of it? ❤️
